cyclid_ui/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:trusty
+
+RUN apt-get update
+RUN apt-get install -qy software-properties-common
+RUN add-apt-repository ppa:brightbox/ruby-ng
+RUN apt-get update
+RUN apt-get install -qy ruby2.3 ruby2.3-dev build-essential cmake mysql-client libmysqlclient-dev libsasl2-dev
+RUN mkdir -p /var/run/cyclid /etc/cyclid /var/log/cyclid /var/lib/cyclid-ui
+COPY files/cyclid-ui-0.2.2.gem /tmp/
```

Comment:
Those are both custom Gems; I want to roll up some other changes and do a patch release of both Cyclid & Cyclid UI; maybe next week? I can open a PR to install from Rubygems once that's done.